build/graph/graph.js

```diff
@@ -279,21 +275,38 @@ async function buildProjectGraph() {
 				return str;
 			});
 
-			let
-				entrySrc;
+			if (content) {
+				let entrySrc;
 
-			if (needLoadStylesAsJS) {
-				entrySrc = path.join(tmpEntries, `${name}.styl.js`);
-				fs.writeFileSync(stylSrc, content);
-				fs.writeFileSync(entrySrc, [webpackRuntime, `require('${stylSrc}');`].join('\n'));
+				if (needLoadStylesAsJS) {
+					entrySrc = path.join(tmpEntries, `${name}.styl.js`);
 
-			} else {
-				entrySrc = stylSrc;
-				fs.writeFileSync(entrySrc, [content, 'generateImgClasses()'].join('\n'));
-			}
+					fs.writeFileSync(stylSrc, content);
+
+					fs.writeFileSync(
+						entrySrc,
+						[webpackRuntime, `require('${stylSrc}');`].join('\n')
+					);
+
+				} else {
+					entrySrc = stylSrc;
+					fs.writeFileSync(
```

Comment:
Многоэтажные конструкции всегда отбиваются переводами для повышения читаемости

```suggestion

					fs.writeFileSync(
```